lms/djangoapps/django_comment_client/base/tests.py

```diff
@@ -347,10 +351,10 @@ def inner(self, default_store, module_count, mongo_calls, sql_queries, *args, **
         return inner
 
     @ddt.data(
-        (ModuleStoreEnum.Type.mongo, 3, 4, 21),
-        (ModuleStoreEnum.Type.mongo, 20, 4, 21),
-        (ModuleStoreEnum.Type.split, 3, 13, 21),
-        (ModuleStoreEnum.Type.split, 20, 13, 21),
+        (ModuleStoreEnum.Type.mongo, 3, 4, 22),
```

Comment:
An individual database query will cost you about 1ms minimum just for latency. If we're adding new features, I think that there's no real way to avoid adding new queries. The big thing we want to avoid are adding queries that blow out with data -- adding a new query for feature X is fine. Adding N queries where N is the number of things the user has for feature X is not.

We should also find ways to offset the additional costs with savings elsewhere. But that's at a more holistic, global view -- i.e. look for the worst things happening across all your transactions and try to improve that, rather than trying to find another query or two to chop off to offset the 1ms added here.